README.md

```diff
@@ -57,6 +57,28 @@ exports.onRouteChange = () => {
 }
 ```
 
+### Customize the scrolling behavior
+```js
+anchorate({ 
+  scroller: function (element) {
+    if (!element) return false
+    element.scrollIntoView({ behavior: 'smooth' })
+    return true
+  }
+})
+```
+
+### Getting results
+```js
+anchorate({ 
+  callback: function (success) {
```

Comment:
After https://github.com/AndersDJohnson/anchorate/pull/2/files#r168865874, this would become:
```
callback: function (err) {
```

Reply:
Done.